### PR TITLE
Insa patch

### PR DESCRIPTION
**Pull request by lpshanley**
:smile:

### CODE REVIEW

**Comment by theVKC** [comment]
Will test shortly. Thank you!

**Comment by lpshanley** [comment]
Let me know if there are any issues and Ill revise it over lunch.

-Lucas
“Valar dohaeris”
#DecimateTheErrorRate
P Please consider the environment before printing this email. P

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 1]
```diff
@@ -1063,24 +1063,51 @@ Function autofill_editbox_from_MAXIS(HH_member_array, panel_read_from, variable_
       End if
     Next
   Elseif panel_read_from = "INSA" then '----------------------------------------------------------------------------------------------------INSA
+    call navigate_to_screen("stat", "insa")
```
Comment (theVKC) [inline MASTER FUNCTIONS LIBRARY.vbs]: I'm going to clear this out, unless you want to @lpshanley. Now, the function does this already at the top. It reduces redundancy in the function.

Comment (lpshanley) [inline MASTER FUNCTIONS LIBRARY.vbs]: Tear it up ☺

-Lucas
“Valar dohaeris”
#DecimateTheErrorRate
P Please consider the environment before printing this email. P

**Comment by theVKC** [comment]
Closed because I pulled it into master instead. Yay for changing scheme mid-month. :)

**Pull request closed without merging**